deployments/fargate/README.md

```diff
@@ -0,0 +1,137 @@
+# AWS Fargate Deployment
+Familiarity with AWS Fargate (Fargate) is assumed. Consult the 
+[User Guide for AWS Fargate](https://docs.aws.amazon.com/AmazonECS/latest/userguide/what-is-fargate.html)
+for further reading.
+
+Unless stated otherwise, it is assumed that the
+[Splunk OpenTelemetry Collector](https://github.com/signalfx/splunk-otel-collector)
+(Collector) is deployed to an ECS Task in a **sidecar** container alongside monitored
+applications.
+
+Applies to Collector release v0.30.0 and above which corresponds to image tag 0.30.0 and above
+in the image repository [here](https://quay.io/repository/signalfx/splunk-otel-collector?tab=tags).
+
+Overview of the container definition for the Collector:
+- Set environment variable `SPLUNK_CONFIG` to the configuration file path. Set `SPLUNK_CONFIG`
+  to the default configuration file path to enable default configuration.
```

Comment:
Does it default to a setting and this is about overriding? It should in which case configuration options.

Reply:
Yes, the default configuration file for the collector container is /etc/otel/collector/gateway_config.yaml but we want the default in fargate to be /etc/otel/collector/fargate_config.yaml.

Reply:
We support setting SPLUNK_CONFIG so getting started and data setup UI should include steps to ensure proper config is used.

Reply:
ok